CHANGELOG.md

```diff
@@ -81,6 +81,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (crypto | x/auth) [#14372](https://github.com/cosmos/cosmos-sdk/pull/18194) Key checks on signatures antehandle.
 * (staking) [#18506](https://github.com/cosmos/cosmos-sdk/pull/18506) Detect the length of the ed25519 pubkey in CreateValidator to prevent panic.
 * (tx) [#18772](https://github.com/cosmos/cosmos-sdk/pull/18772) Remove misleading gas wanted from tx simulation failure log.
+* (x/feegrant) [#18815](https://github.com/cosmos/cosmos-sdk/pull/18815) Steps to modify the 'PeriodReset' field assignment.
```

Comment:
nit, the changelog should be in x/feegrant/CHANGELOG.md and not here.

Reply:
@julienrbrt the modification log records have been adjusted and moved to x/feegrant/CHANGELOG.md. Could you please review it again? Thank you 🙏🙏